### ISSUE TITLE
How do I remove previously added lock?

### ISSUE THREAD

**Issue by x-iLex-x** [jbergler/hass-ttlock#114]
I have added a new lock to TTLock App and it was successfully detected by integration. But then I've removed this lock from TTLock, but I see no way how to remove it from HASS, avoiding reinstallation. I see it in my HASS as unavailable device. Restarting HA or host doesn't help.

Is there any way?

**Comment by grumpyvwman** [comment]
Did you find a way to remove old lock just have same issue after had to replace faulty lock

**Comment by x-iLex-x** [comment]
> Did you find a way to remove old lock just have same issue after had to replace faulty lock

No, unfortunately I didn’t. I have a faulty lock as unavailable device in HA by now.

**Comment by grumpyvwman** [comment]
Thanks for reply I can't seem to find a way to remove either. My lock failed on finger print sensor but was replaced under warranty by simpled door locks
Thanks again

**Comment by jbergler** [comment]
When you click into the unavailable entities settings, is the "delete" option in the bottom left greyed out?

**Comment by x-iLex-x** [comment]
> When you click into the unavailable entities settings, is the "delete" option in the bottom left greyed out?

I only have «empty» device with no entities and no options to remove device.

I have smth like “this device has no entities” when I jump into device.

**Comment by wjwnz** [comment]
I've just replaced a faulty lock, removed it from the App, deleted the integration and added it again.
